src/Wallabag/CoreBundle/Controller/WallabagRestController.php

```diff
@@ -214,8 +213,34 @@ public function getEntriesTagsAction(Entry $entry)
      *       }
      * )
      */
-    public function postEntriesTagsAction(Entry $entry)
+    public function postEntriesTagsAction(Request $request, Entry $entry)
     {
+        $tags = explode(',', $request->request->get('tags'));
+
+        foreach ($tags as $label) {
+            $tagEntity = $this
```

Comment:
In my next commit, I moved this block into a private function of this controller ... I'll have a look to move it into `TagRepository`.